src/System.ComponentModel.Composition.Registration/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      uap;
```

Comment:
Why stop running the tests in uap? I would actually just build the tests as netstandard 2.1 actually, since you said you will provide now an oob implementation so you want to test it everywhere possible.

Reply:
Building test assemblies against netstandard isn't supported anymore, it needs to be a vertical now. I didn't know that uap does support NS21, I noticed that yesterday. I will add it here.